src/templates/base/2019/base.html

```diff
@@ -99,11 +99,134 @@
 {% block contributors %}{{ config.contributors.keys() | length }}{% endblock %}
 
 {% block non_chapter_nav_links %}
-<li><a href="{{ url_for('table_of_contents', year=year, lang=lang) }}">{{ self.table_of_contents_title() }}</a></li>
 <li><a href="{{ url_for('contributors', year=year, lang=lang) }}">{{ self.contributors_title() }}</a></li>
 <li><a href="{{ url_for('methodology', year=year, lang=lang) }}">{{ self.methodology_title() }}</a></li>
 {% endblock %}
 
+{% macro nav_table_of_contents(switcher_name) %}
+  {% if year == '2019' %}
+    {% if switcher_name == "mobile" or switcher_name == "mobile-footer" %}
+      {{ native_table_of_contents_dropdown(switcher_name) }}
+    {% else %}
+      {{ styled_table_of_contents_dropdown(switcher_name) }}
+    {% endif %}
+  {% endif %}
+{% endmacro %}
+
+{% macro native_table_of_contents_dropdown(switcher_name) %}
+{% set current_title = metadata.get('title') if metadata else None %}
+<div class="table-of-contents-switcher">
+  <label for="table-of-contents-switcher-{{switcher_name}}" class="visually-hidden">
+    {{ self.table_of_contents_title() }}
+  </label>
+  <select id="language-switcher-{{switcher_name}}">
+    <option disabled selected value> {{ self.table_of_contents_title() }} </option>
+    <option value="{{ url_for('table_of_contents', year=year, lang=lang) }}">{{ self.foreword_title() }}</option>
```

Comment:
ok i fixed !

Reply:
Doesn't look like this was fixed

Reply:
The title is still set to `self.foreword_title()` instead of `self.table_of_contents_title()`. As I say on Mobile I think we should use `self.table_of_contents_title()` as that's what shows on home page for example.